extras/spoof.md

```diff
@@ -47,6 +47,38 @@ With this, navigate towards `Root -> DeviceProperties -> Add` and add your PCI r
 
 ![](../images/extras/spoof-md/config-gpu.png)
 
+For disabling NVIDIA Kepler you can use these device prperties:
+
+```
+<key>DeviceProperties</key>
+	<dict>
+    <key>PciRoot(0x0)/Pci(0x7,0x0)/Pci(0x0,0x0)/Pci(0x8,0x0)/Pci(0x0,0x0)</key>
+			<dict>
+				<key>AAPL,ndrv-dev</key>
```

Comment:
What is this for?

Reply:
This disables IONVDRV frame buffer:
https://opensource.apple.com/source/IOGraphics/IOGraphics-45.3/IONDRVSupport/IONDRVFramebuffer.cpp.auto.html

When NVIDIA is attached to a PCI slot `AAPL,ndrv-dev`is set true and overrides all AMD connectors so the AMD cards are disconnected from the screens and you can only use remote screen there.
Here we explicitly set `AAPL,ndrv-dev` False and override automatic detection and frame buffer connection so AMD connect to the screens without issue. Every existing property has to be deleted first before it is added again in a different state (here TRUE to FALSE).